plan/config.go

```diff
@@ -36,13 +36,14 @@ const (
 // ReadConfigFromEnviron creates a Config by looking for environment variables
 func ReadConfigFromEnviron() (*Config, error) {
 	const (
-		reportKey         = "REPORT"
-		callTimeoutKey    = "CALL_TIMEOUT"
-		waitForTimeoutKey = "WAIT_FOR_TIMEOUT"
-		waitKey           = "WAIT_FOR"
-		axisKeyPrefix     = "AXIS_"
-		behaviorKeyPrefix = "BEHAVIOR_"
-		jsonReportPathKey = "JSON_REPORT_PATH"
+		reportKey             = "REPORT"
+		callTimeoutKey        = "CALL_TIMEOUT"
+		waitForTimeoutKey     = "WAIT_FOR_TIMEOUT"
+		waitKey               = "WAIT_FOR"
+		axisKeyPrefix         = "AXIS_"
+		behaviorKeyPrefix     = "BEHAVIOR_"
+		behaviorSkipKeyPrefix = "BEHAVIOR_SKIP_"
```

Comment:
Please make this just `"SKIP_"`. Building on the `"BEHAVIOR_"` prefix has the unintended consequence of subtracting from the range of expressible behavior names.